pkg/api/alertmanager.go

```diff
@@ -119,10 +119,19 @@ type GettableSilences = amv2.GettableSilences
 type GettableSilence = amv2.GettableSilence
 
 // swagger:model
-type GettableAlerts amv2.GettableAlerts
+type GettableAlerts = amv2.GettableAlerts
```

Comment:
This is the first time I am seeing `type x = y` syntax :scream: how does it work?

Reply:
Is it equivalent to `type x y`?

Reply:
I can't find the original article I read on the subject which does a better job at explaining than I do, but it is different. The tl;dr is that `type x y` is a type definition, in the eyes of the compiler is a new type just like `type X struct { y }` whereas `type x = y` is type alias which means is just a different name for the same type (and thus would have the same memory footprint).

In practical terms, this allows us to use our types and the returned AM types interchangeably w/o having to do any type casting. We are not operating on these types so it makes sense.